mongo/src/main/java/com/redhat/lightblue/mongo/metadata/MongoMetadata.java

```diff
@@ -283,30 +283,20 @@ public void createNewMetadata(EntityMetadata md) {
                 try {
                     WriteResult result = new InsertCommand(collection, infoObj).executeAndUnwrap();
                     LOGGER.debug("Inserted entityInfo");
-                    String error = result.getError();
-                    if (error != null) {
-                        LOGGER.error("createNewMetadata: error in createInfo: {}" + error);
-                        throw Error.get(MongoMetadataConstants.ERR_DB_ERROR, error);
-                    }
-                } catch (MongoException.DuplicateKey dke) {
+
```

Comment:
Both deprecated (and now removed) and useless.  Error message will be in the exception thrown for an unsuccessful write operation.

http://api.mongodb.org/java/2.14/com/mongodb/WriteResult.html#getError()